packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -103,6 +112,7 @@ const defaultProps = {
   isInvalidMeridiem: false,
   minuteAttributes: {},
   hourAttributes: {},
+  label: '',
```

Comment:
Should we just let this be undefined instead of an empty string?

Reply:
I'll try it.